src/components/pages/AdminHome/AdminCourses.js

```diff
@@ -1,11 +1,78 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import AdminSidebar from './AdminSidebar';
-import '../../../styles/index.less';
+import { connect } from 'react-redux';
+import CourseCard from './AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/coursesActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { Layout, Input } from 'antd';
+const { Search } = Input;
+
+// need to upgrade the courses router/modal to get the instructor name (2 seconds update in modal)
+
+function AdminCourses(props) {
+  const { courses } = props;
+  const dispatch = useDispatch();
+  const idToken = useOktaAuth().oktaAuth.getIdToken();
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
+  }, [dispatch, idToken]);
+
+  const history = useHistory();
+
+  const handleClick = () => {
+    history.push('/admin-add-course');
+  };
+  const onSearch = searchTerm => {
```

Comment:
Same with `Search` let's either remove this or comment that it's still WIP.

Reply:
Handler has been removed! No need for it in the future, the handler will go in the custom search bar file